### ISSUE TITLE
Stacer breaks dnf update or dnf repolist on RHEL 8

### ISSUE THREAD

**Issue by Saladin1812** [oguzhaninan/Stacer#338]
I have tried this on two systems through stacer appimage if I do clean on Stacer it breaks dnf Now when i do sudo dnf update it gives me 
![Screenshot from 2020-01-07 23-42-53](https://user-images.githubusercontent.com/40650341/71918162-bab5d680-31a7-11ea-8589-f0444afa3be4.png)

These lines of unnecessary information. Before that it just updated my repo and installed updates as usual. I did this on RHEL 8 on my PC and problems starts happening. Then I ran dnf update on my other system it behaved normally i suspected that stacer caused the problem on PC so i did the same with my laptop to confirm my doubt and yes it did the same to my laptop now :(

PLEASE DONT USE STACER ON RHEL 8 till this doesnt get fixed, you broke my DNF.

**Comment by jatin-cbs** [comment]
I had to reinstall my distro because of stacer it broke my DNF after i did the system clean. Also tried it on VM.

**Comment by Saladin1812** [comment]
Behaviour before stacer 

https://pastebin.com/9HUKXcNf

Behavior after cleaning with stacer 

https://pastebin.com/77WQBjHy

**Comment by jatin-cbs** [comment]
DNF mainthread logs are so frustrating please help devs you guys broke my dnf

**Comment by Saladin1812** [comment]
I think I found the solution stacer removed a important directory needed on redhat inside /var/logs which is rhsm to store dnf logs.

sudo mkdir -p /var/log/rhsm

This solves the problem

**Comment by Saladin1812** [comment]
Please add this exception in stacer

**Comment by oguzhaninan** [comment]
Hi @jatin-cbs,
You are responsible for the files you delete. Otherwise it may be complicated to define an exception.

**Comment by Saladin1812** [comment]
@oguzhaninan I didnt select any files, it auto cleans the log directory if you check mark that but that on RHEL specificaly includes the rhsm file its just important. On other distros cleaning logs is fine just RHEL has an exception.

**Comment by Saladin1812** [comment]
stacer has a checkmark for deleting application logs in its clean feature. That feature should atleast work on all distros and not cause any problem just a suggestion but else is your wish

**Comment by oguzhaninan** [comment]
@jatin-cbs  You are right. I didn't think so. Thank you for feedback. I'm gonna fix this in the new version.
